script/Ciel/Day 13/13_01_01_CIEL13_5B.txt

```diff
@@ -1078,27 +1085,28 @@ But in the end, it was all meaningless.
 [sha:206a81e5a3a4857b8b13a5a2e30178a4db3b5265]{
 -- Page 641, Offset 35111.
 -- “夜が明けない”と、だれかが、ようやく口にして。
-After all this time, someone finally shouts out, "There is no dawn".
+After all this time, somebody finally shouts 'Dawn isn't coming!'.
 }
 [sha:3f2a4a554b4d5f9d38b88bbca774b4cd60525300]{
 -- Page 641, Offset 35112.
 -- ひとりひとり、丹念に、でも何の愛着もなく■された。
-One by one, with diligence, but without compassion, she ■ us all.
+One by one, she ■■■■■ all of us painstakingly yet coldheartedly.
 }
 [sha:2d969d41645e8d62c59911af0a8a1b9627ed83aa]{
 -- Page 642, Offset 35113.
 -- ……いちじかんにひとりぐらいのペースだったから、たぶん、みっかぐらいたったあとのはなしです。
-...She went at a pace of around one person every hour, so it was likely only three days that have passed.
+// i love hiragana!
```

Comment:
it's _scriptura continua_ and it's actually 2smart4u literary genius and not at all unintelligible nonsense